barrister/runtime.py

```diff
@@ -5,6 +5,8 @@
     :license: MIT, see LICENSE for more details.
 """
 
+from twisted.internet import defer
```

Comment:
adding this here, one needs to add this to the dependencies (setup.py / requirements.txt)